packages/kit/src/packaging/test/fixtures/assets/jsconfig.json

```diff
@@ -3,8 +3,8 @@
 		"checkJs": true,
 		"baseUrl": ".",
 		"paths": {
-			"$lib/*": ["src/lib/*"]
+			"$lib/*": ["./src/lib/*"]
 		}
 	},
-	"include": ["src/**/*.d.ts", "src/**/*.js", "src/**/*.svelte"]
+	"include": ["./src/**/*.d.ts", "./src/**/*.js", "./src/**/*.svelte"]
```

Comment:
IIUC the `./` prefix is unnecessary if you specify `baseUrl`

Reply:
Oh you're right, yes. They are unnecessary then.

Reply:
Just to be clear, you want me to add `baseUrl` everywhere instead? TypeScript 4.6 will enforce one or the other

Reply:
I'm saying that in cases where there's already a `baseUrl` (directly or via `extends`) there's no need to add `./` prefixes